tools/odbc/test/CMakeLists.txt

```diff
@@ -26,6 +26,10 @@ add_executable(
   tests/col_attribute/col_attribute.cpp
   tests/col_attribute/char_query.cpp
   tests/col_attribute/int_query.cpp
+  tests/col_attribute/DATE_query.cpp
+  tests/col_attribute/TIME_query.cpp
+  tests/col_attribute/TIMESTAMP_query.cpp
+  tests/col_attribute/INTERVAL_query.cpp
```

Comment:
Nit: Can these filenames match the case of the other filenames.

Reply:
They do reference SQL data types and SQL convention is to upper case SQL keywords like data types....